src/lualib/BooleanClass.ts

```diff
@@ -0,0 +1,17 @@
+import { __TS__Boolean } from "../lualib/Boolean";
+
+export class Boolean {
```

Comment:
I don't really see the added value of providing these as actual classes, I think it will just cause confusing bugs, especially since the equality comparisons will be broken. I think I would prefer to just translate `new Boolean(x)` the same as `Boolean(x)`.